css/media-queries.md

```diff
@@ -164,9 +164,9 @@ Since we have everything 1080px width and less specified, we’ll add something
 	}
 }
 ```
-	- The green media query no longer applies because styles that come after will override any styles that come earlier in the css file.  The red rule is applying to *everything* wider than 1081px, including 1200px and up.
-	- The browser reads css and js from top to bottom, so if it applies different styles to the same screen width, *the one that is applied last wins*.
-	- When the media query styles you expect are not appearing, overriding is a common culprit.
+- The green media query no longer applies because styles that come after will override any styles that come earlier in the css file.  The red rule is applying to *everything* wider than 1081px, including 1200px and up.
```

Comment:
Please change these back, as they are are notes about the above code, and not a new section.